test/jdk/java/nio/file/attribute/BasicFileAttributeView/CreationTime.java

```diff
@@ -81,7 +81,7 @@ static void test(Path top) throws IOException {
             System.out.println("creationTime.toMillis() == " + creationTime.toMillis());
             // If the file system doesn't support birth time, then skip this test
             if (creationTime.toMillis() == 0) {
```

Comment:
Is this still useful? After https://bugs.openjdk.org/browse/JDK-8338696 we'd never get the epoch (`0`). If birth time is not supported, the last modified time is being returned. I think this can go and the `SkippedException` branch removed as well.

Reply:
Thanks for your correction, the `SkippedException` for epoch (0) has been deleted.